scripts/swift-version.sh

```diff
@@ -34,7 +70,9 @@ find_xcode_for_swift() {
     return 1
 }
 
-if [[ "$REALM_SWIFT_VERSION" ]]; then
+if [[ "$REALM_XCODE_VERSION" ]]; then
```

Comment:
I would generally use `-n` to be certain, but I think that this will work in any event.